Bynder/Api/Converters/LowercaseConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Bynder.Models;
+
+namespace Bynder.Api.Converters
+{
+    public class LowerCaseConverter : ITypeToStringConverter
+    {
+        public bool CanConvert(Type typeToConvert)
+        {
+            return typeof(AssetType?).IsAssignableFrom(typeToConvert);
```

Comment:
One last comment, can we create this Converter to be generic and not specific to AssetType, so it works for all objects? The converter should call the .toString() method of the object and then .ToLower.